packages/node-client/bin/proxy-multi-part-form.js

```diff
@@ -4,23 +4,22 @@ const args = process.argv.slice(2);
 
 const nango = new Nango({ host: 'http://localhost:3003', secretKey: args[0] });
 
-const pdf = fs.readFileSync('./test.pdf', 'utf8');
+const filePath = './test.pdf';
+const fileName = filePath.split('/').pop();
+const buffer = fs.readFileSync(filePath);
 const formData = new FormData();
-formData.append('content', pdf);
-
-console.log(formData);
+formData.append('file', new Blob([buffer]), fileName);
 
 nango
     .proxy({
         providerConfigKey: 'unauthenticated',
         connectionId: 'u',
         baseUrlOverride: 'http://localhost:3009',
         method: 'POST',
-        endpoint: '/',
+        endpoint: '/upload/multipart',
```

Comment:
maybe I should commit the python webserver I used to received the uploaded file. Let me know what you think

Reply:
I think we need a bigger initiative to help test more stuff than just this, a test website would be nice